pyjokes/jokes_eu.py

```diff
@@ -5,7 +5,6 @@
     'Zer esaten dio bit batek besteari? Busean ikusten gara!',
     'Zer da terapeuta bat? - 1024 Gigapeuta',
     'Aita, aita, aita!!! Utziko al didazu telebista iksten? Bai! Baina piztu ez!',
-    'Matematika liburu batek bere buruaz beste egin zuen...zergatik? Problema asko zituelako.'
```

Comment:
A Math book that suicides because having too many problems... I don't see the problem in this joke.